models/workday__position_overview.sql

```diff
@@ -34,8 +34,9 @@ position_data_enhanced as (
         position_data.compensation_grade_code,
         position_data.compensation_grade_profile_code
     from position_data
-    left join position_job_profile_data on position_job_profile_data.position_id = position_data.position_id
-    where not position_data._fivetran_deleted
+    left join position_job_profile_data 
+        on position_job_profile_data.position_id = position_data.position_id
```

Comment:
Request to change the format of the left join

Reply:
Sorry, what do you mean by change the format?

Reply:
On the `main` branch the left join is all on a single line

https://github.com/fivetran/dbt_workday/blob/513f6f55ca04b479890b35a35684658740784656/models/workday__position_overview.sql#L37

This suggestion is to update the format to my change above where the left join is broken out into two lines
```sql
    left join position_job_profile_data 
        on position_job_profile_data.position_id = position_data.position_id
```

Reply:
Ah. Updated.